src/boot/api.js

```diff
@@ -1,7 +1,7 @@
 import { Api, JsonRpc } from 'eosjs'
 import axios from 'axios'
 
-const signTransaction = async function (actions) {
+const signTransaction = async function (actions, options) {
```

Comment:
@evgenibir  so here you could change to `async function (actions, options = DEFAULT_OPTIONS )` or where DEFAULT_OPTIONS is the value that you've been copy / pasting

Reply:
fixed